docs/user-guide/dream/dream-data-reduction.ipynb

```diff
@@ -211,48 +354,30 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "17",
+   "id": "21",
    "metadata": {},
    "outputs": [],
    "source": [
     "workflow[TwoThetaBins] = sc.linspace(\n",
-    "    dim=\"two_theta\", unit=\"rad\", start=0.8, stop=2.4, num=17\n",
+    "    dim=\"two_theta\", unit=\"rad\", start=0.8, stop=2.4, num=201\n",
     ")"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "18",
+   "id": "22",
    "metadata": {},
    "outputs": [],
    "source": [
-    "grouped_dspacing = workflow.compute(IofDspacingTwoTheta)\n",
+    "grouped_dspacing = workflow.compute(IofDspacingTwoTheta).bins.concat(\"event_time_zero\")\n",
```

Comment:
Does this mean that the result has a new dimension?

Reply:
Yes, we expect NeXus data to have the `event_time_zero` dimension, because it records events from more than one pulse.
The data we have in the geant4/mcstas files isn't very realistic, it has values for tof that exceed 71ms.

Reply:
But shouldn't the time have been removed at some point during conversion to d-spacing?